tools/release-tools/setVersion.js

```diff
@@ -146,6 +151,68 @@ function parseArgs() {
     return true;
 }
 
+function updateVersions() {
+    const rootVersion = require(process.cwd() + "/package.json");
+    let newVersion = calculateVersion(rootVersion.version);
+    if (newVersion) {
+        console.log("New version [" + theVersion.release + "] => [" + newVersion + "]");
+        if (!isReact && !isReactNative) {
+            theVersion.release = newVersion;
+        }
+    }
+
+    let packages = theVersion.pkgs = theVersion.pkgs || {};
```

Comment:
why is there two "= theVersion.pkgs"

Reply:
This is so that if theVersion.pkgs is currently undefined it will get assigned, this is the same as saying
```
if (!theVersion.pkgs) {
   theVersion.pkgs = {};
}

let packages = theVersion.pkgs;
```
or maybe a little more clearer as 
```
theVersion.pkgs = (theVersion.pkgs || {});
let packages = theVersion.pkgs;
```